include/rocksdb/statistics.h

```diff
@@ -529,6 +529,9 @@ enum Tickers : uint32_t {
   // Footer corruption detected when opening an SST file for reading
   SST_FOOTER_CORRUPTION_COUNT,
 
+  FILE_READ_CORRUPTION_RETRY_COUNT,
```

Comment:
nit: Can we add some documentation to this counter for which type of file retry read this includes?